pkg/kopia/repository/connect_or_create.go

```diff
@@ -40,7 +41,7 @@ func ConnectToOrCreateKopiaRepository(
 	case err == nil:
 		// If repository connect was successful, we're done!
 		return nil
-	case kerrors.IsInvalidPasswordError(err):
+	case kerrors.IsInvalidPasswordError(err): // TODO: Verify
```

Comment:
Verify what?

Reply:
Verify that matching is not broken. I've added test for error matchers.